### ISSUE TITLE
Array that is provided in place of a ParamArray parameter is treated as Params argument

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn#8570]
For example, the the argument `New Integer() {  }` in the following code is reported as a Argument operation with kind ParamArray

``` vb
Public Sub M0(ParamArray b As Integer())
End Sub

M0(New Integer() {  })
```

It should be treated as simple argument instead.

@JohnHamby This is the intended behavior, correct?

**Comment by genlu** [comment]
This is fixed by #18494.
